src/app/shared/services/user-session.service.ts

```diff
@@ -44,16 +44,13 @@ export class UserSessionService implements OnDestroy {
       repeatLatestWhen(this.userProfileUpdated$),
       switchMap(auth => {
         if (!auth.authenticated) return of<UserProfile | undefined>(undefined);
-        return this.currentUserService.getProfileInfo().pipe(
-          catchError(_e => {
-            this.userProfileError$.next();
-            return EMPTY;
-          })
-        );
+        return this.currentUserService.getProfileInfo().pipe(retry(1));
       }),
       distinctUntilChanged(), // skip two undefined values in a row
     ).subscribe(profile => {
       this.session$.next(profile ? { user: profile } : undefined);
+    }, () => {
+      this.userProfileError$.next();
     });
```

Comment:
`}, () => {` should not be at the same indent as `).subscribe(profile => {` as it is an arg of subscribe.
Usually I write subscription with error like that:
```
).subscribe(
  () => {...},
  () => {... },
)
```
Here, as it is one instructions it should even be: 
```
).subscribe(
  () => this.session$.next(profile ? { user: profile } : undefined),
  () => this.userProfileError$.next(),
)
```